README.md

```diff
@@ -88,7 +88,7 @@ expressGAuth({
   clientDomain: 'http://localhost:5555',
   allowedDomains: ['reaktor.fi', 'reaktor.com'], // User needs to login with Google and email with these domains.
   allowedEmails: ['john@example.com', 'jussi@example.com'], // These users are allowed login through Google auth.
-  publicEndPoints: ['/logout'], // These end points do not require any authentication.
+  publicEndPoints: ['/logout'], // These end points do not require any authentication. Before validating if request matches this list, the url parameters are stripped away
```

Comment:
To be really safe, this new functionality could be enabled with a separate config option (which would be disabled by default). Even though there might not be anyone affected by the change, this is technically a breaking change. (If someone is using `https://what.ever.domain?foo=bar` as a public endpoint, that will not work anymore with these changes.)

Reply:
That is very much true!  Let me add that separate config option.